lib/home/home_screen.dart

```diff
@@ -125,7 +125,7 @@ class HomeScreen extends StatelessWidget {
     final navBar = NavBar.of(context);
 
     return Scaffold(
-      body: navBar.screen,
+      body: SafeArea(bottom: false, child: navBar.screen),
```

Comment:
The `AppBar` has a built-in safe area so it doesn't overlap the device UI overlay. Hopefully with this `SafeArea` here, we won't need `AppBar`s for the home screens.